draft-packages/tag/KaizenDraft/Tag/Tag.tsx

```diff
@@ -60,7 +75,7 @@ const Tag = (props: TagProps) => {
   return (
     <div
       className={classNames(styles.root, {
-        [styles.default]: variant === "default",
+        [styles.default]: variant === "default" || "profile",
```

Comment:
This will always be a `truthy` expression. If the goal is to use this style when `variant` is either `default` or `profile`, this would need to be `variant === "default" || variant === "profile"`.

Reply:
Thank you !! silly me